shared/chat/inbox/container.js

```diff
@@ -19,30 +20,34 @@ const getFilter = (state: TypedState) => state.chat.get('inboxFilter')
 
 const createImmutableEqualSelector = createSelectorCreator(defaultMemoize, I.is)
 
-const passesFilter = (i: Constants.InboxState, filter: string): boolean => {
+const passesStringFilter = (filter: string, toCheck: string): boolean => {
+  return toCheck.toLowerCase().indexOf(filter.toLowerCase()) >= 0
+}
+
+const passesParticipantFilter = (participants: I.List<string>, filter: string, you: ?string): boolean => {
```

Comment:
we need to filter ourselves out as we don't appear in the list of participants and its weird for that to match